src/util/object_forwarding.rs

```diff
@@ -233,3 +233,16 @@ pub(super) fn forwarding_bits_offset_in_forwarding_pointer<VM: VMBinding>() -> O
 pub(super) fn forwarding_bits_offset_in_forwarding_pointer<VM: VMBinding>() -> Option<isize> {
     unimplemented!()
 }
+
+pub(crate) fn debug_get_object_forwarding_info<VM: VMBinding>(object: ObjectReference) -> String {
+    let forwarding_bits = get_forwarding_status::<VM>(object);
```

Comment:
Again before actually checking the forwarding information, we should check if the metadata is mapped.  Some VMs (such as CRuby) are using side forwarding bits.  It is possible that `object` is invalid and its metadata is not mapped, yet.